.github/workflows/upload-release-to-plugin-repo.yml

```
@@ -248,3 +248,41 @@ jobs:
                   svn import "$VERSION" "$PLUGIN_REPO_URL/tags/$VERSION" -m "Committing version $VERSION" \
                   --no-auth-cache --non-interactive  --username "$SVN_USERNAME" --password "$SVN_PASSWORD" \
                   --config-option=servers:global:http-timeout=300
+
+    sync-assets:
+        name: Sync assets
+        runs-on: ubuntu-latest
+        environment: wp.org plugin
+        if: ${{ github.event.release.assets[0] }}
+        env:
+            PLUGIN_REPO_URL: 'https://plugins.svn.wordpress.org/gutenberg'
+            STABLE_VERSION_REGEX: '[0-9]\+\.[0-9]\+\.[0-9]\+\s*'
+            SVN_USERNAME: ${{ secrets.svn_username }}
+            SVN_PASSWORD: ${{ secrets.svn_password }}
+            VERSION: ${{ github.event.release.name }}
```

Unused env var, and I expect we'll also end up replacing `VERSION` with a commit sha if we update on commits to trunk.

```suggestion
            PLUGIN_REPO_URL: 'https://plugins.svn.wordpress.org/gutenberg'
            SVN_USERNAME: ${{ secrets.svn_username }}
            SVN_PASSWORD: ${{ secrets.svn_password }}
            VERSION: ${{ github.event.release.name }}
```

---

Using SHA per https://github.com/WordPress/gutenberg/pull/68052/commits/6514ea0bea170ec7fe75afb477ff80bc4b1780f7.